src/netspresso_trainer/loggers/base.py

```diff
@@ -56,6 +56,7 @@ def __init__(
         self._result_dir = result_dir
         OmegaConf.save(config=self.conf, f=(result_dir / "hparams.yaml"))
 
+        self.use_mlflow: bool = getattr(self.conf.logging, "mlflow", False)
```

Comment:
I think it would be better to remove `getattr`.
Unlike other fields, making an exception (allow optional) for this does not seem appropriate.

```suggestion
        self.use_mlflow: bool = self.conf.logging.mlflow
```